.github/workflows/pr.yaml

```diff
@@ -68,6 +68,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        lxd_channel: ["5.0/stable", "latest/edge"]
```

Comment:
FYI, 256 runners can be used in a single workflow run according to github actions docs